toucan_connectors/redshift/redshift_database_connector.py

```diff
@@ -312,6 +322,15 @@ def get_status(self) -> ConnectorStatus:
             self.check_port(self.host, self.port)
         except Exception as e:
             return ConnectorStatus(status=False, details=self._get_details(1, False), error=str(e))
+
+        # Basic db query
+        try:
+            self._retrieve_tables(database=self.default_database)
+        except Exception as e:
+            return ConnectorStatus(
+                status=False, details=self._get_details(3, False), error=e.args[0]
+            )
+
```

Comment:
Maybe we should do a simple connect like postgres connector here : 

```suggestion

        # Basic db query
        try:
            redshift_connector.connect(**self.get_connection_params(database=database_name))
        except Exception as e:
            return ConnectorStatus(
                status=False, details=self._get_details(3, False), error=e.args[0]
            )

```

Reply:
we have to implement get_connection_params accordignly for redshift

Reply:
Did you mean something like that : https://github.com/ToucanToco/toucan-connectors/pull/616/commits/0aa0d96c0651a6aae818b73e61e67744c6c7e146 ?